### PR TITLE
Add PercentageDiscount

### PR DESCRIPTION
**Pull request by artursmet**

### CODE REVIEW

File: prices/tests.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 import unittest
 
 from prices import (FixedDiscount, History, LinearTax, Price, PriceRange,
-                    inspect_price)
+                    inspect_price, PercentageDiscount)
```
Comment (patrys) [inline prices/tests.py]: Please run `isort` on this file :)

File: prices/__init__.py [hunk 2]
```diff
@@ -324,5 +324,26 @@ def apply(self, price_obj):
                      currency=price_obj.currency, history=history)
 
 
+class PercentageDiscount(PriceModifier):
```
Comment (patrys) [inline prices/__init__.py]: What if it was called `FractionDiscount` and did not explicitly multiply the factor by `0.01`?

**Comment by patrys** [comment]
I'd like to rename `FractionDiscount` to `FractionalDiscount`.

Can we also test `repr`?

Also currently `PercentageDiscount` does not have any specific properties so it could be replaced by a function that returns a `FractionalDiscount` instance.

**Comment by artursmet** [comment]
@patrys Refactored, now percentage_discount is a function, instead of class

**Pull request merged**